server/gamemodes/chess.py

```diff
@@ -256,7 +256,9 @@ def validate_king_move(
         """Validate a king's move."""
         absolute_file_delta = abs(file - king.file)
         absolute_rank_delta = abs(rank - king.rank)
-        if (not absolute_rank_delta) and not king.has_moved:
+        #if statement to handle where castling is attempted
+        if (not absolute_rank_delta) and not king.has_moved and absolute_file_delta > 1:
+            #check that castling is valid
```

Comment:
```suggestion
        # Check for castling attempt.
        if (not absolute_rank_delta) and not king.has_moved and absolute_file_delta > 1:
```
Comments should describe what code does, not how it does it - we can see it's an if statement.